Controller/CRUDController.php

```diff
@@ -456,6 +473,8 @@ public function editAction($id = null, Request $request = null)
                 $templateKey = 'preview';
                 $this->admin->getShow();
             }
+        } else {
+            $session->set($versionKey, $this->admin->getVersion($object));
```

Comment:
Having this version number in the form will avoid some possible weird behavior with session.